url4df616fce9_s3e0n1g1_prototype3_tgz-pJ8-Trading_Engine_MainJ8/src/Trading_Engine/myDatabase.java

```diff
@@ -0,0 +1,571 @@
+package Trading_Engine;
+
+import java.io.*;
+import java.sql.*;
+import java.util.LinkedList;
+
+public class myDatabase {
+	private static Connection connection;
+	private int totalMatch;
+
+	public myDatabase(){
+		try {
+			Class.forName("org.sqlite.JDBC");
+		} catch (Exception e) {
+			System.out.println("Could not find JDBCdriver : " + e);
+		}
+		try {
+			connection = DriverManager.getConnection("jdbc:sqlite:test.db");
+		} catch (Exception e) {
+			System.out.println("Could not establish connection : " + e);
+		}
+		deleteAllTables();
+	}
+	public static void insertAllDatabase (Connection con, BufferedReader br, String newHead, String dbName, LinkedList<String> overviewResult){
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 11. By comparison, 98% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.